docs/src/examples/components/List/Content/ListExampleEndMedia.tsx

```diff
@@ -4,18 +4,21 @@ import { List } from '@stardust-ui/react'
 const ellipsis = <span>&hellip;</span>
 
 const ListExample = () => (
-  <List>
+  <List selection>
```

Comment:
👍

Reply:
Cannot make endMedia example selectable because of the issue - https://github.com/stardust-ui/react/issues/370